### DISCUSSION TITLE
No code to see after 'create and push' with VS 2022

### DISCUSSION THREAD

**Discussion by ghost** [community/community#36489] [New to GitHub]
I created a project and uploaded it to my github account using create and push with VS 2022. However, I don't see any code there in the code tab. I only see the 'how to start' overview in github.
What is wrong there?

Thanks in advance
Guido
![code](https://user-images.githubusercontent.com/116038457/196298553-fb00dc58-d919-4495-8a99-957562be5926.png)

**Comment by theOther0** [comment]
I am also new to git and github. I don't know what you mean by using VS 2022 to create and push to your github account.

However, from what I've learned you need to connect your local git to the remote server of the github .  This [site ](https://www.notion.so/zarkom/Introduction-to-GitHub-202af6f64bbd4299b15f238dcd09d2a7#636208f914614ba5b5fd64bc7701e342)would be helpful to learn.

**Reply by ghost** [reply]
There is an integrated module in VS 2022 that allows you to push your projects directly to Github. Normally, I see the master or main version directly in the status bar of VS 2022. But I don't see anything. That's probably why I don't see any code on the website. With VS 2019, everything worked fine.

**Comment by ossanna16** [comment]
Hey there @Kewarra and thank you for asking a good question! I'm glad you found our community 🙂
